src/include/migraphx/op/pointwise.hpp

```
@@ -45,14 +45,19 @@ struct pointwise
         {
             MIGRAPHX_THROW("should have one submodule.");
         }
-        auto* pm    = mods.front();
+        auto* pm = mods.front();
+        if(pm->get_output_shapes().size() != 1)
+            MIGRAPHX_THROW("submodule should have only one output.");
+        // if pointwise op doens't have any input args (const-foldable) then just return output
+        // shape
+        if(inputs.empty())
+        {
+            return pm->get_output_shapes().front();
+        }
```

This should throw an error.

---

Done